src/tests/ftest/daos_vol/bigio.py

```diff
@@ -15,6 +16,7 @@ class DaosVol(VolTestBase):
     :avocado: recursive
     """
 
+    @skipForTicket("DAOS-6654")
```

Comment:
Should we be skipping this? I'm curious if there is a reason why it wasn't skipped initially: https://daosio.atlassian.net/browse/DAOS-6654

Reply:
I read the comments in the ticket, but not sure why it wasn't skipped initially either.

I reported this failure in DAOS-10198, but Scott said this is the known issue reported in DAOS-6654, so I added this tag.

Reply:
In that case, when this PR lands, we should probably leave a comment on DAOS-6654 to unskip the test when fixed.

Reply:
I created a ticket to remove the tag. Thanks for the reminder.